templates/powershell/build/TagRelease.ps1

```diff
@@ -0,0 +1,79 @@
+<#
+.SYNOPSIS
+Set Tag for repository
+.DESCRIPTION
+Set Tag for repository
+
+.PARAMETER AppVersion
+Mandatory. Application version
+.PARAMETER PSHelperDirectory
+Mandatory. Directory Path of PSHelper module
+.EXAMPLE
+.\TagRelease.ps1 -AppVersion <AppVersion>
+#> 
+
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory)]
+    [string] $AppVersion
+)
+
+Set-StrictMode -Version 3.0
+
+[string]$functionName = $MyInvocation.MyCommand
+[datetime]$startTime = [datetime]::UtcNow
+
+[int]$exitCode = -1
+[bool]$setHostExitCode = (Test-Path -Path ENV:TF_BUILD) -and ($ENV:TF_BUILD -eq "true")
+[bool]$enableDebug = (Test-Path -Path ENV:SYSTEM_DEBUG) -and ($ENV:SYSTEM_DEBUG -eq "true")
+
+Set-Variable -Name ErrorActionPreference -Value Continue -scope global
+Set-Variable -Name InformationPreference -Value Continue -Scope global
+
+if ($enableDebug) {
+    Set-Variable -Name VerbosePreference -Value Continue -Scope global
+    Set-Variable -Name DebugPreference -Value Continue -Scope global
+}
+
+Write-Host "${functionName} started at $($startTime.ToString('u'))"
+Write-Debug "${functionName}:AppVersion=$AppVersion"
+
+try {
+
+    $exists = git tag -l "$AppVersion"
+    if ($exists) { 
+        Write-Host "Tag already exists"
+    }    
+    git tag $AppVersion --force
```

Comment:
use the PS-Helps Invoke-command for all the git commands?

Reply:
Fixed